pkg/generator/ingress_translator.go

```diff
@@ -107,18 +106,12 @@ func (translator *IngressTranslator) translateIngress(ctx context.Context, ingre
 				}
 
 				endpoints, err := translator.endpointsLister.Endpoints(split.ServiceNamespace).Get(split.ServiceName)
-				if apierrors.IsNotFound(err) {
-					translator.logger.Warnf("Endpoints '%s/%s' not yet created", split.ServiceNamespace, split.ServiceName)
-					break
-				} else if err != nil {
+				if err != nil {
```

Comment:
This will now manifest in immediate reconcile retries, is that what we want? Don't we want to be triggered by endpoints/service events if we can't find something?

Reply:
Yes, we want to be triggered by endpoints/services events if it was not found.
I reverted the last change but sorry if I misunderstood it.